docs/http.rst

```diff
@@ -487,3 +495,63 @@ Remove a given ChannelID subscription from a UAID.
 **Return Codes:**
 
 See :ref:`errors`.
+
+Report ACKs and NAKs
+~~~~~~~~~~~
+
+Report the processing status of messages for a given UAID.
+
+Because connections can be at a premium, only "batch" mode is
+supported.
+
+**Call:**
+
+.. http:put:: /v1/naks/{uaid}
+
+::
+
+   Authorization: Bearer {auth_token}
+
+**Parameters:**
+
+A JSON block containing the status of messages, divided by "ACK"
+(success) and "NAK" (failure). To save bandwidth, currently, only
+NAKs are recorded.
+
+For instance:
+
+.. code-block:: json
+
+    {"nak":[{"id": {Message ID}, "ver": {Message Version},
+             "code": {Return Code}, "msg": {Error Message}},
+            ...
+           ],
+     "ack":[{"id": {Message ID}, "ver": {Message Version}},
+            ...
+           ]
+    }
+
+
+For this example:
+
+    :id: The message ID included in the notification data.
+
+    :ver: The message version field included in the notification data.
+
+    :code: An optional return code indicating the type of error encountered. Error codes may be specific to individual bridge implementations.
```

Comment:
That reminds me; I think we still need to document the codes that the client sends (https://dxr.mozilla.org/mozilla-central/rev/c67dc1f9fab86d4f2cf3224307809c44fe3ce820/dom/push/PushServiceWebSocket.jsm#41-59).